### ISSUE TITLE
Animated banner is not using correct colors

### ISSUE THREAD

**Issue by broccolini** [primer/design#35]
Looks like something went wrong between after effects and shipping the animation, the Primer blue's are all dulled and not rendering correctly. It's possible this is on the after effects export end, or may be something we can fix in code.

cc @ashygee

**Comment by jhuashao** [comment]
this could be a small issue with on the export side of illustrator. At times illustrator will flip to cmyk and auto convert lots of colors.

**Comment by ashygee** [comment]
@broccolini I just opened the illustrator file and @jhuashao is correct that all of the colors shifted slightly. I'm not entirely sure why as the document is set to an RGB colorspace.

**Illustrator file linked in animation AE file**
<img width="835" alt="screenshot 2019-02-26 11 07 33" src="https://user-images.githubusercontent.com/10384315/53439423-d7458780-39b6-11e9-9e13-271ad8b0f22a.png">


I also exported the svg from Figma and opened in Illustrator and there was no color shifting with this. The svg is correctly displaying `blue-600` as `#005CC5` so I'm not quite sure what happened originally.

**Illustration opened from Figma svg export**
<img width="667" alt="screenshot 2019-02-26 11 08 14" src="https://user-images.githubusercontent.com/10384315/53439442-e298b300-39b6-11e9-9564-7428042d7021.png">


I will see if I can update the colors in the Illustrator file and if that fixes the issue.
